src/web/lib/export.js

```diff
@@ -0,0 +1,102 @@
+import JSZip from "jszip";
```

Comment:
This is a pretty big package (https://bundlephobia.com/result?p=jszip@3.1.5) that will be used by a small fraction of the users loading the website. Could we try dynamically importing it when `performThemeExport` is called?

Reply:
I forgot that Webpack makes this pretty easy, so [this little tweak should do import `performThemeExport` and dependencies on demand](https://github.com/mozilla/FirefoxColor/pull/494/files#diff-462b9ae37a974155d575e827e27c7076R144)